command/role.go

```diff
@@ -10,7 +10,7 @@ import (
 )
 
 const (
-	roleUsage = "role [number]"
+	roleUsage = "role [number/name/prefix of name]"
```

Comment:
I'm not sure if mentioning the prefix helps or adds confusion here.

Reply:
what else do you suggest?